Netkan/Sources/Github/GithubApi.cs

```diff
@@ -100,14 +121,79 @@ public List<GithubUser> getOrgMembers(GithubUser organization)
             );
         }        
 
-        private string Call(string path)
+        /// <summary>
+        /// Download a URL via the GitHubAPI.
+        /// Will use a token if we have one.
+        /// </summary>
+        /// <param name="url">The URL to download</param>
+        /// <returns>
+        /// null if the URL isn't on GitHub, otherwise the contents of the download
+        /// </returns>
+        public string DownloadText(Uri url)
+        {
+            string ghOwner  = null;
+            string ghRepo   = null;
+            string ghBranch = null;
+            string ghPath   = null;
+            if (TryGetGitHubPath(url, out ghOwner, out ghRepo, out ghBranch, out ghPath))
```

Comment:
If you want to save some lines, you can also do inline declaration:
```suggestion
            if (TryGetGitHubPath(url, out string ghOwner, out string ghRepo, out string ghBranch, out string ghPath))
```

Reply:
That's a nice language feature!

Reply:
It is!